newIDE/app/src/ResourcesList/LocalResourceSources.js

```diff
@@ -192,6 +197,50 @@ export default [
   },
 ];
 
+// function _loadValuesFromLocalStorage(): ?PreferencesValues {
```

Comment:
Unfortunately this is not scalable, as this code is redundant with the implementation already existing in PreferencesProvider. We can't allow ourselves do put such code in every file that would need the preferences. Imagine the nightmare when you want to change the storage.
The editor should also not assume that things are written in the local storage... it might be saved in memory, on disk, or even some other exotic stuff in the future. 
That's why we have a `PreferencesContext` that React components can use to get the method to load or save a preference.
Using this context, they can manipulate the preferences - and they don't care how it's done.

Here we're not in a file using React though - but can you think about a way to use the methods from PreferencesContext? :)

Reply:
Ok, I will try to think over it. Maybe I can create load/save functions in PreferencesContext and export them and use them in both PreferencesProvider and other files needed. 🤔

Reply:
That's one solution, though ideally all your functions would come from the context itself - so that if later we use a second PreferencesProvider (that could happen, for example, a set of preference per user, etc...), the good one is automatically picked up by the context.
See if you can pass the functions from a React component to the load/save functions.

Reply:
I have updated the PR, please review it.